### PR TITLE
Added references to the external themes wiki page in README

### PR DESCRIPTION
**Pull request by dritter**
Added references to the `external themes` wiki page.

### CODE REVIEW

**Comment by dritter** [comment]
Hmm.. Three mentions of one wiki page in the same file.. Not the best solution, but if it helps to minimize the amount of pull requests.. Hopefully, this is all right.

File: README.markdown [hunk 1]
```diff
@@ -122,6 +124,8 @@ If you have many functions that go well together, you can put them as a `abcyzea
 
 If you would like to override the functionality of a plugin distributed with Oh My Zsh, create a plugin of the same name in the `custom/plugins/` directory and it will be loaded instead of the one in `plugins/`.
 
+Feel free to add your theme to the [wiki](https://github.com/robbyrussell/oh-my-zsh/wiki/External-themes), if you want to share it with others.
```
Comment (ncanceill) [inline README.markdown]: I believe this reference is not really necessary, which would cut it down to two.

**Comment by ncanceill** [comment]
Also could you please squash your commits to keep the history clean and atomic?

**Comment by dritter** [comment]
@ncanceill All done. 
Now there are just two mentions of the "external themes" wiki page and this is just one commit.

**Comment by ncanceill** [comment]
Neat!

**Pull request closed without merging**